madrid/bicimad.py

```diff
@@ -0,0 +1,144 @@
+import requests
+import pandas as pd
+
+from madrid.auth_credentials import auth_bicimad
```

Comment:
As you say, when need to probably put this as an environment variable but it's ok for now.